crates/bevy_crevice/README.md

```diff
@@ -50,7 +50,7 @@ uniform MAIN {
 ```
 
 ```rust
-use crevice::std140::{AsStd140, Std140};
```

Comment:
Can you add a note to the readme that this is a fork of the crevice maintained by bevy.

Reply:
Yup it's in the "remaining", but I would like Cart to chime in on how he wants to present that, or at least someone with better English